src/app/locales/pt_BR.ts

```diff
@@ -0,0 +1,538 @@
+  'Animations Tests': {
+    action:
+      'Se você usar animações e testes de unidade, adicione `mods[1].NoopAnimationsModule` à chamada de` TestBed.initTestEnvironment`.',
+  },
+  DefaultIterableDiffer: {
+    action: 'Não use `DefaultIterableDiffer`,` KeyValueDiffers#factories` ou `IterableDiffers#factories.`',
+  },
+  'Template Tag': {
+    action: 'Renomeie as tags `template` para` ng-template`.',
+  },
+
+  OpaqueToken: {
+    action: 'Substitua `OpaqueToken` por` InjectionToken`.',
+  },
+  DifferFactory: {
+    action: 'Se você usar `DifferFactory.create (...)` remova `ChangeDetectorRef` como um argumento.',
+  },
+  'ErrorHandler Parameter': {
+    action: 'Não passe nenhum argumento para o construtor ErrorHandler',
+  },
+  ngProbeToken: {
+    action:
+      'Se você usar o ngProbeToken. certifique-se de importá-lo de `@ angular/core` em vez de` @angular/platform-browser`',
+  },
+  TrackByFn: {
+    action: 'Use TrackByFunction em vez de TrackByFn',
+  },
+  'i18n Pipe Change': {
+    action:
+      'Se você usar os pipes `date, `currency, `decimal ou `percentage`, na v5 você verá algumas pequenas mudanças no formato. Para aplicativos que usam `locales diferentes de en-us, você precisará importá-lo e, opcionalmente, `locale_extended_fr` de `@angular/common/i18n_data/locale_fr` e registerLocaleData(local).',
+  },
+  gendir: {
+    action:
+      'Não confie no uso de `gendir`, em vez disso use` skipTemplateCodeGen`. <a href=https://github.com/angular/angular/issues/19339#issuecomment-332607471" target="_blank"> Leia mais </a>',
+  },
+  'Dynamic ngUpgrade': {
+    action:
+      'Substitua `downgradeComponent`,` downgradeInjectable`, `UpgradeComponent` e` UpgradeModule` importados de `@ angular/upgrade`. Use a nova versão em `@angular/upgrade/static`',
+  },
+  'Animations in Core': {
+    action:
+      'Se você usar serviços para animações ou ferramentas de @angular/core, você deve importá-los de @angular/animations',
+  },
+  ngOutletContext: {
+    action: 'Substitua `ngOutletContext` por` ngTemplateOutletContext`.',
+  },
+  collectionChangeRecord: {
+    action: 'Substitua `CollectionChangeRecord` por` IterableChangeRecord`',
+  },
+  Renderer: {
+    action: 'Em todos os lugares que você estiver usando o Renderer, substitua-o por Renderer2',
+  },
+  'Router Query Params': {
+    action: 'Se você utilizar `preserveQueryParams`, substitua por ´queryParamsHandling` .',
+  },
+  Http: {
+    action:
+      'Se você usar o antigo módulo `HttpModule` e o serviço` Http`, substitua-o pelo serviço` HttpClientModule` e `HttpClient`. HttpClient melhora a ergonomia por padrão (você não precisa mais usar um `map` para converter a resposta para JSON) e agora suporta tipos de retorno e interceptores. Você pode ler mais em [angular.io] (https://angular.io/guide/http).',
+  },
+  'DOCUMENT in @angular/platform-browser': {
+    action: 'Se você usar DOCUMENT de `@angular/platform-browser`, deve começar a importá-lo de `@angular/common`',
```

Comment:
`Se você está usando ...` or `Se você usa ...` can be more effective in this case

Reply:
Done